### ISSUE TITLE
Doing nothing in gnome 3.8

### ISSUE THREAD

**Issue by thorus90** [darkxst/multiple-monitor-panels#23]
I try to install it via extensions.gnome.org and chromium asks me if I want to install it but after accepting it nothing happens. Under ~/.local/share/gnome-shell/extensions gets no directory created. Is there a log file to look at it?

Can I install it manually ?

Thanks

**Comment by jchardis2** [comment]
You probably have similar problems that I did in gnome 3.8. Read the comment left by korline at https://extensions.gnome.org/extension/323/multiple-monitor-panels/ . I had to reboot my computer and my extensions showed up with an error. Thats what I did but others left other possible solutions ex read Xyon's solution.

Korus:
This extension work with gnome 3.8, but not out of the box. If you want to make it working, you have to edit a file to force the activation in gnome-shell 3.8 - Download the source code at github here : https://github.com/darkxst/multiple-monitor-panels/archive/master.zip - Extract it in your personnal folder - Move the folder extra-panels@darkxst.feathertop.org in ~/.local/share/gnome-shell/extensions/ - then in this folder edit the metadata.json and add your gnome-shell version at the end of the line "shell-version" - Restart gnome-shell (alt+F2, type r and Enter) This should work. Of course, if you have already the extension installed but inactive because of gnome-shell upgrade, you juste have to edit the metadata.json and restart gnome-shell. NB : This method is just a workaround, it seems to work, but it's not sure it's fully compatible.

**Comment by thorus90** [comment]
Yeah I have gnome 3.8.4, and got it working with that workaround. Thanks for that. Despite I'm only getting the top panel on the second monitor (Activities, active app, language, sound settings, clock, and menue) and not the bottom part with the open windows. Is this as designed or is it not working properly?

**Comment by jchardis2** [comment]
I really wish I could help you bc it is the same way with me and I haven't used it without this fix. I haven't looked into it any further, but if you do, please let me know!
